lib/pos_consensus.go

```diff
@@ -384,14 +384,17 @@ func (fc *FastHotStuffConsensus) HandleLocalVoteEvent(event *consensus.FastHotSt
 // HandleValidatorVote is called when we receive a validator vote message from a peer. This function processes
 // the vote locally in the FastHotStuffEventLoop.
 func (fc *FastHotStuffConsensus) HandleValidatorVote(pp *Peer, msg *MsgDeSoValidatorVote) error {
+	glog.Infof("FastHotStuffConsensus.HandleValidatorVote: Received vote msg %s", msg.ToString())
```

Comment:
can we change this to be `glog.V(2).Infof` actually? I'm worried that this will get very noisy in production.